### PR TITLE
Fix missing link from safely drain node

### PR DESCRIPTION
**Pull request by tengqm**
Closes: #3879

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6816)

### CODE REVIEW

**Comment by heckj** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/tasks/administer-cluster/safely-drain-node.md, line 37 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/website/6816#-L2GpffHF4w84qbTDUkZ:-L2GpffHF4w84qbTDUk_:bto1cpn) ([raw file](https://github.com/kubernetes/website/blob/03d3ff0e680399b083361395d20092414c665961/docs/tasks/administer-cluster/safely-drain-node.md#L37)):*
> ```Markdown
> hardware maintenance, etc.). Safe evictions allow the pod's containers
> to
> [gracefully terminate](/docs/concepts/containers/container-lifecycle-hooks/) and
> ```

I get what you're doing with making the link to the lifecycle details, but there is nothing in that page that really references the underlying topic of graceful termination. In the absence of any specific documentation on that, I think this may be better without a corresponding link at all. 

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/website/6816)*

**Comment by tengqm** [comment]
@heckj Changed the link to pod termination, where "graceful" termination is mentioned.

File: docs/tasks/administer-cluster/safely-drain-node.md [hunk 1]
```diff
@@ -33,9 +33,8 @@ This task assumes that you have met the following prerequisites:
 You can use `kubectl drain` to safely evict all of your pods from a
 node before you perform maintenance on the node (e.g. kernel upgrade,
 hardware maintenance, etc.). Safe evictions allow the pod's containers
-to
-[gracefully terminate](/docs/tasks/#lifecycle-hooks-and-termination-notice) and
-will respect the `PodDisruptionBudgets` you have specified.
+to [gracefully terminate](https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods)
```
Comment (zhangxiaoyu-zidif) [inline docs/tasks/administer-cluster/safely-drain-node.md]: could you change it to relative path, `/docs/...`

**Comment by tengqm** [comment]
@zhangxiaoyu-zidif revised.

**Pull request merged**